src/styles/search-results.css

```diff
@@ -0,0 +1,21 @@
+.image-gallery {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.card-image {
+  height: 200px;
+  width: 200px;
+  object-fit: cover;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 10px;
+  margin-right: 10px;
+  border-radius: 8px;
+}
+
+.no-results {
+  font-size: larger;
+  margin: auto;
```

Comment:
Defining margin like this is the same as
```
margin-top: auto;
margin-right: auto;
margin-bottom: auto;
margin-left: auto;
```
But `margin: auto` only works for the horizontal axis so youre declaring things you dont need here. If you want to keep it in shorthand form you can just change this to:
```
margin: 0 auto;
```